demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -2418,3 +2421,73 @@ def test_IsTestsSectionValidValidator_obtain_invalid_content_items():
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def test_is_command_or_script_name_starts_with_digit_invalid():
+    """
+    Given
+    - One invalid integration with one command starting with a digit character.
+    When
+    - Calling the IsCommandOrScriptNameStartsWithDigitValidator obtain_invalid_content_items function.
+    Then
+    - Make sure one failure is returned and the error message is correct.
+    """
+    with ChangeCWD(REPO.path):
+        content_items = [
+            create_integration_object(
+                paths=["script.commands"],
```

Comment:
let's also add a valid command here and ensure it's not mentioned.

Reply:
Added a valid command alongside the invalid one and made sure it does not appear in the error message